src/storage/src/hummock/compactor/fast_compactor_runner.rs

```diff
@@ -313,7 +313,8 @@ impl CompactorRunner {
             split_weight_by_vnode: task.split_weight_by_vnode,
             use_block_based_filter: true,
         };
-        let factory = StreamingSstableWriterFactory::new(context.sstable_store.clone());
+        let factory = UnifiedSstableWriterFactory::new(context.sstable_store.clone());
```

Comment:
We can change the [logic ](https://github.com/risingwavelabs/risingwave/blob/e6d76a2cd5d0a469ab5a50c50c998f93db827f5d/src/storage/src/hummock/compactor/mod.rs#L144) in normal compaction runner as well.

Reply:
Yes, will do it in next pr.